content/doc/developer/views/read-only.adoc

```diff
@@ -107,6 +113,72 @@ Removing controls from a page, like an X button on a repeatable element:
 </j:if>
 ----
 
+== Allowing access to a view
+
+Normally in Jenkins the layout tag of a page will have a permission
+attribute which governs which permission is required to access the page. 
+
+Jenkins has a concept of 'impliedBy' in it's permission, which means that each permission is allowed to be automatically granted by another one.
+_Job/ExtendedRead_ is implied by _Job/Configure_, which means that requiring _Job/ExtendedRead_
+to access a view, will also grant _Job/Configure_ access.
+
+Here are some examples that show what you will need to update the layout tag to:
+
+Requiring the _Job/ExtendedRead_ permission in Jelly:
+
+[source,xml]
+----
+<l:layout permission="${it.EXTENDED_READ}">
+...
+</l:layout>
+----
+
+_Overall/SystemRead_ Jelly:
+
+[source,xml]
+----
+<l:layout permission="${app.SYSTEM_READ}">
+...
+</l:layout>
+----
+
+_Overall/SystemRead_ Groovy:
+
+[source,groovy]
+----
+l.layout(permission: app.SYSTEM_READ) {
+...
+}
+----
+
+== Server side changes required
+
+Firstly you will need to update any server side permission checks to now check
+on load for the _Read_ permission, rather than the _Configure_  or _Administer_ permission.
```

Comment:
Should reference https://www.jenkins.io/doc/developer/security/read-access/#using-staplerproxy